edi_oca/static/src/js/widget_edi.esm.js

```diff
@@ -23,8 +24,14 @@ export class EdiConfigurationWidget extends Component {
         );
         this.action.doAction(action);
     }
+
+    getInfo() {
+        return this.props.record.data.edi_config;
```

Comment:
```suggestion
        return this.props.record.data[this.props.name];
```
This way it uses the field